operators/src/source/ogr_source.rs

```diff
@@ -469,25 +491,40 @@ where
     ) -> Result<()> {
         // TODO: add OGR time filter if forced
         let dataset = Self::open_gdal_dataset(&dataset_information)?;
-        let mut layer = dataset.layer_by_name(&dataset_information.layer_name)?;
-        let use_ogr_spatial_filter = dataset_information.force_ogr_spatial_filter
-            || layer.has_capability(gdal::vector::LayerCaps::OLCFastSpatialFilter);
-
-        if use_ogr_spatial_filter {
-            // rectangular geometry from West, South, East and North values.
-            let filter_geometry = query_rectangle.spatial_bounds.try_into()?;
-            // TODO: log uses spatial filter
-            // TODO: use OGR_L_SetSpatialFilterRect() once GDAL crate supports it
-            // NOTE: the OGR-filter may be inaccurately allowing more features that should be returned in a "strict" fashion.
-            layer.set_spatial_filter(&filter_geometry);
-        }
+
+        let mut use_ogr_spatial_filter = false;
+
+        let mut features_provider = if let Some(sql) = dataset_information.sql_query.as_ref() {
+            // TODO: spatial filter
```

Comment:
the spatial_filter in OGR execute_sql is a Geometry. Should I build one from the query rectangle? It would be more efficient to filter against a rectangle (min/max x/y) but that would require a manual rewrite on the query and not sure if it works with postgis index.

Reply:
4a9b3290ef5f38a4ef0f8c00ff7d5b1dda51ffb9